common/v2/features/SendAssets/SendAssets.tsx

```diff
@@ -63,9 +65,11 @@ function SendAssets() {
   const goToPrevStep = () => setStep(Math.max(0, step - 1));
   const goToFirstStep = () => setStep(0);
 
+  const goBack = () => (step === 0 ? history.push('/') : goToPrevStep());
```

Comment:
Prefer path from https://github.com/MyCryptoHQ/MyCrypto/blob/gau/common/v2/config/routePaths.ts#L125. For example `ROUTE_PATHS.ROOT.path` or `ROUTE_PATHS.DASHBOARD.path` instead of hardcoded path.

Reply:
Thanks! I just did the same as in ReceiveAssets, but I see your point.

Changed it in both of them.

Reply:
Awesome, thanks. We are trying to use paths from config for future code; unfortunately there is still a lot of old code which should be changed eventually.